src/network/clientpackethandler.cpp

```diff
@@ -1587,9 +1587,9 @@ void Client::handleCommand_MediaPush(NetworkPacket *pkt)
 {
 	std::string raw_hash, filename, filedata;
 	u32 token;
-	bool cached;
+	bool ephemeral;
```

Comment:
I would prefer not renaming this

Reply:
I'd be fine with not renaming in the client, but then I would want to rename it in the server (where the field id called "ephemeral").  I think the field should have the same name at both ends.

The reason I chose to rename in the client is to match the documentation.